core/server/src/main/java/alluxio/master/AlluxioMasterRestServiceHandler.java

```diff
@@ -0,0 +1,323 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the “License”). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.master;
+
+import alluxio.Configuration;
+import alluxio.Constants;
+import alluxio.MasterStorageTierAssoc;
+import alluxio.Version;
+import alluxio.master.block.BlockMaster;
+import alluxio.underfs.UnderFileSystem;
+import alluxio.util.CommonUtils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import com.codahale.metrics.Counter;
+import com.codahale.metrics.Gauge;
+import com.codahale.metrics.MetricRegistry;
+import com.qmino.miredot.annotations.ReturnType;
+
+import java.io.IOException;
+import java.util.Comparator;
+import java.util.Map;
+import java.util.Set;
+import java.util.SortedMap;
+import java.util.TreeMap;
+
+import javax.annotation.concurrent.NotThreadSafe;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+/**
+ * This class is a REST handler for requesting general master information.
+ */
+@NotThreadSafe
+@Path(AlluxioMasterRestServiceHandler.SERVICE_PREFIX)
+@Produces(MediaType.APPLICATION_JSON)
+// TODO(cc): Investigate auto-generation of REST API documentation.
+public final class AlluxioMasterRestServiceHandler {
+  private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
+  private static final String ALLUXIO_CONF_PREFIX = "alluxio";
+
+  public static final String SERVICE_PREFIX = "master";
+  public static final String GET_RPC_ADDRESS = "rpc_address";
```

Comment:
I don't think all these 'GET' prefix are needed, since they are already reflected by the @GET annotation.